packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -1408,6 +1427,16 @@ class _NestedScrollPosition extends ScrollPosition implements ScrollActivityDele
     ));
   }
 
+  /// Update the [ignorePointer] property of the [ScrollContext].
+  /// This is used to allow interaction with the scrollable's content.
+  /// When a scrollable reaches overscrolled state, e.g. when using [BouncingScrollPhysics],
+  /// the expected behavior is that the user can interact with the content and
+  /// not have to wait for the scroll to settle.
+  @override
+  void updateIgnorePointer(bool ignorePointer) {
+    context.setIgnorePointer(ignorePointer);
+  }
```

Comment:
Is it not possible to just call position.context.setIgnorePointer? The updateIgnorePointer/setIgnorePointer is a bit confusing

Reply:
I'll take a look at it.